cartridges/LINK_adyen/cartridge/client/default/js/adyen-cse.js

```diff
@@ -16,6 +16,79 @@ $('button[value="submit-payment"]').on('click', function (e) {
     encryptedDataValue = cseInstance.encrypt(cardData);
     encryptedData.val(encryptedDataValue);
     $('#cardNumber').val("");
+
+    if($('#selectedPaymentOption').val() == "Adyen" && $('#directoryLookup').val() == "true" && !$("input[name='brandCode']:checked").val()) {
+        $('#requiredBrandCode').show();
+        return false;
```

Comment:
do we need to return it with false ? could this not be removed.

Reply:
Yes, otherwise the next page will be shown

Reply:
clear